code/script.js

```diff
@@ -227,99 +399,182 @@ const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
+// This function to start (and restart) the game, resets guess counter + timer
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  numberOfGuesses = 0
+  totalGuesses.innerText = 0
```

Comment:
Good job with resetting both the countert and the guesses in the start function.